python/bindings/openravepy_planningutils.cpp

```diff
@@ -92,11 +92,12 @@ object pyGetReverseTrajectory(PyTrajectoryBasePtr pytraj)
 
 void pyVerifyTrajectory(object pyparameters, PyTrajectoryBasePtr pytraj, dReal samplingstep, bool releasegil=true)
 {
+    PlannerBase::PlannerParametersConstPtr parameters = openravepy::GetPlannerParametersConst(pyparameters);
     openravepy::PythonThreadSaverPtr statesaver;
     if( releasegil ) {
         statesaver.reset(new openravepy::PythonThreadSaver());
     }
-    OpenRAVE::planningutils::VerifyTrajectory(openravepy::GetPlannerParametersConst(pyparameters), openravepy::GetTrajectory(pytraj),samplingstep);
+    OpenRAVE::planningutils::VerifyTrajectory(parameters, openravepy::GetTrajectory(pytraj),samplingstep);
```

Comment:
@cielavenir don't you also have the same problem with `openravepy::GetTrajectory`?

Reply:
the background is, currently GetTrajectory just returns member variable and does not call py::extract, so it is not required but I accept your suggestion to avoid further surprise after changing GetTrajectory implementation.

cc @yoshikikanemoto @rolandjitsu